src/ln-service.d.ts

```diff
@@ -1 +1,3 @@
 declare module "ln-service"
+
+type primitive = string | boolean | number
```

Comment:
Any reason why this should be here and not in `types.ts`?

Reply:
I actually thought this was a global types file. My bad. Will fix.

I think we need to move all type definitions to a global .d.ts file though. I'll do that in a separate PR.